.github/workflows/ci.yml

```diff
@@ -10,6 +10,14 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.10"
-      - run: python -m pip install -r requirements.txt
-      - run: dbt deps
-      - run: dbt build
+          cache: 'pip'
+      - name: Install dependencies
+        run: |
+          python -m pip install -r requirements.txt
+          pipx install meltano && meltano install
+      - name: Run Extract and Load
+        run: meltano run tap-jaffle-shop target-duckdb
+      - name: Install dbt Dependencies
+        run: dbt deps
+      - name: dbt Build
+        run: dbt build
```

Comment:
These are still identical as before, except I added step names for improved readability in CI history.